lib/ansible/config/manager.py

```diff
@@ -382,7 +382,7 @@ def get_config_value(self, config, cfile=None, plugin_type=None, plugin_name=Non
         except AnsibleError:
             raise
         except Exception as e:
-            raise AnsibleError("Unhandled exception when retrieving %s:\n%s" % (config), orig_exc=e)
+            raise AnsibleError("Unhandled exception when retrieving %s" % config, orig_exc=e)
```

Comment:
i would actually add the exception message to_native(e) instead, which was my original intention